R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -1656,6 +1656,18 @@ test_that("greatest() and least() on a DataFrame", {
   expect_equal(collect(select(df, least(df$a, df$b)))[, 1], c(1, 3))
 })
 
+test_that("input_file_name()", {
+  path <- tempfile(pattern = "input_file_name_test", fileext = ".txt")
+  write.table(iris[1:50, ], path, row.names = FALSE, col.names = FALSE)
+
+  df <- read.text(path)
```

Comment:
does it work with `df <- read.json(jsonPath)`?

if yes, consider adding to the test for column functions

(again, this is regarding: https://github.com/apache/spark/pull/17817 to consolidate/skip tests)